src/Models/Scopes/Product/WithProductStockScopeMsi.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Rapidez\Msi\Models\Scopes\Product;
+
+use Illuminate\Database\Eloquent\Builder;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Scope;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Schema;
+
+class WithProductStockScopeMsi implements Scope
+{
+    public function apply(Builder $builder, Model $model)
+    {
+        $stockId = config()->has('rapidez.stock_id') ? config('rapidez.stock_id') : $this->getInventoryStockId();
```

Comment:
maybe `$stockId = config('rapidez.stock_id') ?? $this->getInventoryStockId();` is enough? 🤔

Reply:
Actually the `config()` helper should have a fallback attribute available so
`$stockId = config('rapidez.stock_id', $this->getInventoryStockId());` should suffice 🙂

Reply:
@indykoning Changed to use fallback from config function itself.